### PR TITLE
api code to fetch the routes

### PR DESCRIPTION
**Pull request by Pankhuri2909**
updated code
@sandeepchitturi
@pabbati2000

### CODE REVIEW

File: src/main/java/com/jetblue/task/fetch/route/dao/RouteDao.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.jetblue.task.fetch.route.dao;
+
+import com.jetblue.task.fetch.route.model.Route;
+
+import java.util.List;
+
+public interface RouteDao {
```
Comment (pabbati2000) [inline src/main/java/com/jetblue/task/fetch/route/dao/RouteDao.java]: Normally we are not creating interface for Dao layer in JetBlue projects

File: src/main/java/com/jetblue/task/fetch/route/model/Route.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        return seasonal;
+    }
+
+    public void setSeasonal(String seasonal) {
+        this.seasonal = seasonal;
+    }
+
+    @Override
+    public String toString() {
+        return "Route{" +
```
Comment (pabbati2000) [inline src/main/java/com/jetblue/task/fetch/route/model/Route.java]: You could try using ToStringBuilder utility class from apache commons, you may just want to try it.

File: src/main/java/com/jetblue/task/fetch/route/dao/RouteDao.java (hunk 1 above)
Comment (Pankhuri2909) [reply src/main/java/com/jetblue/task/fetch/route/dao/RouteDao.java]: okay

**Review by pabbati2000 — APPROVED** [review]

File: src/main/java/com/jetblue/task/fetch/route/model/Route.java (hunk 2 above)
Comment (Pankhuri2909) [reply src/main/java/com/jetblue/task/fetch/route/model/Route.java]: Done

**Pull request closed without merging**